### PR TITLE
fixed regex on plates

### PR DESCRIPTION
**Pull request by tozaicevas**
strange bug, didn't let me change the original list in foreach

### CODE REVIEW

**Comment by UndeadRat22** [comment]
Change PlateAPIResult to class instead.

File: backend/Epicenter.Domain/Services/DTO/Plate/Responses/PlateAPIResponse.cs [hunk 1]
```diff
@@ -9,8 +9,17 @@ public class PlateAPIResponse
 
         public void UpdateMatchesPattern(string pattern)
         {
+            List<PlateAPIResult> newResults = new List<PlateAPIResult>();
             Regex regex = new Regex(pattern);
-            Results.ForEach(result => result.MatchesPattern = regex.IsMatch(result.Plate));
+            Results.ForEach(result =>
```
Comment (UndeadRat22) [inline backend/Epicenter.Domain/Services/DTO/Plate/Responses/PlateAPIResponse.cs]: change PlateAPIResult to class instead.

Comment (tozaicevas) [reply backend/Epicenter.Domain/Services/DTO/Plate/Responses/PlateAPIResponse.cs]: done

File: backend/Epicenter.Domain/Services/DTO/Plate/Responses/PlateAPIResult.cs [hunk 2]
```diff
@@ -1,7 +1,8 @@
 ﻿namespace Epicenter.Domain.Services.DTO.Plate.Responses
 {
-    public struct PlateAPIResult
+    public class PlateAPIResult
```
Comment (UndeadRat22) [inline backend/Epicenter.Domain/Services/DTO/Plate/Responses/PlateAPIResult.cs]: wow, what a great idea! :DD

**Pull request merged**